README.md

```diff
@@ -50,7 +50,7 @@ Another is to place an SSL termination EC2 instance in front of the ELBs.
 
 ## feed-dns
 
-`feed-dns` manages a Route53 hosted zone, updating entries to point to the correct ELBs. It is designed to be run as a
+`feed-dns` manages a Route53 hosted zone, updating entries to point to ELBs or CNAMEs of your choosing. It is designed to be run as a
```

Comment:
Wording is slightly wrong. CNAMEs point to other records, rather than pointing to a CNAME. Should be something like "updating entries to point to ELBs or arbitrary hostnames".

Reply:
Agreed. Hostname is a better name.